engine/lib/phx/src/system/guid.rs

```diff
@@ -2,17 +2,21 @@ use std::sync::atomic::{AtomicU64, Ordering};
 
 static NEXT_ID: AtomicU64 = AtomicU64::new(1);
 
-#[no_mangle]
-pub extern "C" fn GUID_Create() -> u64 {
-    NEXT_ID.fetch_add(1, Ordering::Relaxed)
+#[repr(C)]
```

Comment:
This is not needed since guid doesn't have any fields and is just a marker struct.